### PR TITLE
resolves #1. Add basic server and register plugin

### LINKED ISSUE #1: create basic server in example folder
basic hapi server needed to show how plugin can be used

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Resolves #1 
Linked to #2 

Add basic server. No tests yet.

### CODE REVIEW

**Comment by JMurphyWeb** [comment]
Ready to review

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (SimonLab) [inline .gitignore]: we might also need to add npm-debug.log* 
We will add the other values when we add more features

File: index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+const register = function (server, options, next) {
+  console.log('registering tags');
```
Comment (SimonLab) [inline index.js]: console.log is helpful for debuging but you should delete them on production code
see the point 3 of https://codequs.com/p/Syo7zYOtle/10-things-you-should-do-while-running-node-js/

File: .gitignore (hunk 1 above)
Comment (JMurphyWeb) [inline .gitignore]: Done on next PR

File: index.js (hunk 2 above)
Comment (JMurphyWeb) [inline index.js]: Done on next PR

**Pull request merged**